packages/office-ui-fabric-react/src/components/Facepile/Facepile.Props.ts

```diff
@@ -76,6 +76,11 @@ export interface IFacepilePersona extends React.HTMLProps<HTMLButtonElement | HT
   onMouseOut?: (ev?: React.MouseEvent<HTMLElement>, persona?: IFacepilePersona) => void;
 
   /**
+   * If provided, the handler will be called when the component is mounted or unmounted
+   */
+  refHandler?: (el?: HTMLElement, persona?: IFacepilePersona) => void;
+
```

Comment:
How did you add the ref? I was able to add it to the persona through getPersonaProps.

```
      getPersonaProps: (persona: IFacepilePersona) => {
        if (extraDataType === ExtraDataType.name) {
          return {
            imageShouldFadeIn: this.state.imagesFadeIn,
            hidePersonaDetails: false,
            ref: (c) => console.dir(c)
          };
        } else if (extraDataType === ExtraDataType.stats) {
          return {
            imageShouldFadeIn: this.state.imagesFadeIn,
            hidePersonaDetails: false,
            primaryText: `[${persona.data}]`,
            ref: (c) => console.dir(c)
          };
        }
        return {
          imageShouldFadeIn: this.state.imagesFadeIn,
          ref: (c) => console.dir(c)
        };
```